src/ray/gcs/gcs_client/gcs_client.cc

```diff
@@ -134,6 +134,8 @@ void GcsClient::Disconnect() {
   }
 }
 
+bool GcsClient::IsConnected() { return gcs_rpc_client_.get() != nullptr; }
```

Comment:
what if gcs_rpc_client_ is null?

Also what does connected mean here? If it's connected, but due to the network issue, some how it can't be connected, what should that return?

Do you mean IsInited here?

Reply:
gcs_client has a method called `Connect()`, `IsConnected()` means whether this method is called or not. I can add a comment here to make it clear.

Reply:
Hmm this makes me feel like we are exposing impl detail to the API (from the comment). Maybe we should write "Connect means the client has an open connection to GCS. Client can be connected via `Connect` and disconnected via `Disconnect`. All API calls will be ignored if it is false" or something like that?